lib/ood_core/job/adapters/linux_host/launcher.rb

```diff
@@ -60,7 +60,11 @@ def start_remote_session(script)
     cmd = ssh_cmd(submit_host(script), ['/usr/bin/env', 'bash'])
 
     session_name = unique_session_name
-    output = call(*cmd, stdin: wrapped_script(script, session_name))
+    ssh_host_delimited = ssh_hosts.to_a.join(':')
+    environment = {"ssh_host" => ssh_host_delimited}
```

Comment:
When setting env vars here lets use the convention of uppercase:

```
environment = {"SSH_HOSTS" => ssh_hosts.to_a.join(':')}
```

It is also okay to have this be one line since `ssh_hosts.to_a.join(':')` is very readable as is.

Reply:
Also is there a problem with space delimiting the value? That way you could omit the line in the bash script where you substitute `:` for `" "`

Reply:
Yes on second thought, Open3.capture3 supports spaces in their env variables (unlike FFI)